apps/delivery-options/src/views/MyParcelDeliveryOptions/Pickup/PickupLocationList/PickupLocationCarrierGroup.vue

```diff
@@ -78,8 +78,7 @@ const {
     return locationCode.value === option.value;
   },
 });
-
 const price = computed(() => {
-  return getDeliveryTypePrice(DeliveryTypeName.Pickup, props.carrier);
+  return getDeliveryTypePrice(DeliveryTypeName.Pickup, props.options[0]?.carrier);
```

Comment:
waarom uit de opties en niet gewoon uit carrier?

Reply:
Bedoel je dan `props.carrier`? want die bevat alleen de carriername en niet de identifier.

Reply:
Dan is dat waar de fout zit, daar zou de carrier identifier in moeten zitten, die heeft altijd meer informatie dan de carrier name

Reply:
Ik heb het aangepast zodat nu de carrier identifier in props.carrier staat.